### PR TITLE
Advantage design changes

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Updated the advantage and livepatch pages to match the design feedback.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage and http://0.0.0.0:8001/livepatch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that design is happy

### CODE REVIEW

**Comment by anasereijo** [comment]
### /advantage

- [ ] demo doesn't show the changes you did next to me. Tried to update the demo and looks the same

### /livepatch - Looks good 👍

**Comment by anthonydillon** [comment]
@anasereijo @matthewpaulthomas Sorry should be there now.

**Comment by anasereijo** [comment]
@anthonydillon awesome

File: templates/advantage/index.html [hunk 1]
```diff
@@ -19,19 +19,19 @@ <h1>Ubuntu Advantage for Infrastructure</h1>
...
 
   <div class="u-fixed-width">
-    <hr class="p-separator" style="margin: 2rem 0;">
+    <hr class="p-separator">
   </div>
 
   <div class="row">
     <div class="col-6 u-sv3">
-      <h4>Your personal subscription</h4>
+      <h2>Your personal subscription</h2>
```
Comment (matthewpaulthomas) [inline templates/advantage/index.html]: As shown in the wireframes, this should be “Your free personal subscription”. I’m sorry I didn’t update the copydoc to match until this morning.

File: templates/advantage/index.html [hunk 2]
```diff
@@ -19,19 +19,19 @@ <h1>Ubuntu Advantage for Infrastructure</h1>
 <section class="p-strip" style="overflow:visible;">
```
Comment (matthewpaulthomas) [inline templates/advantage/index.html]: It needn’t block landing, but: The intro section is so tall that, on my screen, without scrolling I can’t even see whether I’m signed in. This seems suboptimal for a page that people call a “dashboard”. Is there a simple way to reduce the height? It’s using a lot of height for padding and angles.

File: templates/advantage/index.html [hunk 3]
```diff
@@ -58,7 +58,7 @@ <h4>Your personal subscription</h4>
```
Comment (matthewpaulthomas) [inline templates/advantage/index.html]: –1: the detach command doesn’t work.

> `ubuntu-advantage: error: unrecognized arguments: {token was here}`

The correct command is just `ua detach` without anything following.

File: templates/advantage/index.html [hunk 4]
```diff
@@ -282,28 +284,32 @@ <h4>EAL2</h4>
...
-        <p class="u-no-margin--bottom"><a href="/login" class="p-button--neutral">Register</a> to get your free subscription.</p>
-      </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <h2>Free for personal use</h2>
+      <p>Anyone can use UA Infrastructure Essential for free on up to 3 machines (limitations apply).  All you need is an Ubuntu One account. And if you’re a recognised <a href="https://wiki.ubuntu.com/Membership" class="p-link--external">Ubuntu community member</a>, it’s free on up to 50 machines.</p>
```
Comment (matthewpaulthomas) [inline templates/advantage/index.html]: –1: This contradicts the static text above, which says “You are entitled to use Ubuntu Advantage Essential on 3 machines” whether you’re an Ubuntu community member or not.

Either the community entitlement is included in this release, in which case that text should query whether you’re a community member and show “3” or “50” as appropriate.

Or the community entitlement is _not_ included in this release, in which case the “And if you’re a recognised Ubuntu community member…” sentence here should be omitted for now.

**Comment by anthonydillon** [comment]
@anasereijo @matthewpaulthomas happy?

**Comment by anasereijo** [comment]
when logged in the ESM and Livepatch links are too close

Screenshot 2019-09-10 at 16 16 57

**Comment by matthewpaulthomas** [comment]
As we just discussed F2F, remove the “You are entitled to use Ubuntu Advantage Essential on 3 machines” until there’s an API for determining whether it’s 3 or 50.

I’m sorry I didn’t notice this before, but this is pretty much a contradiction, depending on the definition of the word “about”:

image

This can easily be fixed by dropping the words “about buying an enterprise subscription.”

**Comment by anthonydillon** [comment]
@anasereijo @matthewpaulthomas are you happy with this now?

File: templates/advantage/index.html [hunk 5]
```diff
@@ -5,34 +5,33 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1AW8wQu1Uvq0xFtd_c2JwNdic075MPOTx7k2KHqFPPeo/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip--suru is-dark is-deep">
+
```
Comment (hatched) [inline templates/advantage/index.html]: extra spacing seems excessive ;)

File: templates/advantage/index.html (hunk 1 above)
Comment (anthonydillon) [reply templates/advantage/index.html]: Done

File: templates/advantage/index.html (hunk 2 above)
Comment (anthonydillon) [reply templates/advantage/index.html]: Done and signed off with design.

File: templates/advantage/index.html (hunk 3 above)
Comment (anthonydillon) [reply templates/advantage/index.html]: Done, removed the token from detach.

File: templates/advantage/index.html (hunk 4 above)
Comment (anthonydillon) [reply templates/advantage/index.html]: We don't know if they are a member so I will remove the text.

File: templates/advantage/index.html (hunk 5 above)
Comment (anthonydillon) [reply templates/advantage/index.html]: Removed :+`

**Pull request merged**